source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,245 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+ - Updated email templates to the new branding.
+
+#### Packaging Changes
+ - Updated in-product strings referencing E10 & E20 to new packaging.
+ - SAML has been moved from the highest level tier (Enterprise - former E20) to mid-level (Professional) while Certificate-based authentication feature had no change.
+ - E20, Professional, and Enterprise license SKUs are now supported for installing Enterprise plugins.
+
+#### Beta features Promoted to General Availability
+   - Archived channels
+   - Compliance exports
+   - Custom terms of service
+   - Guest Accounts
+   - System Roles
+   - Plugins
+
+#### Permalink Previews
+ - Added support for permalink previews for posts in Mattermost. Previews are generated to minimize context switching when sharing message links in Channels.
+
+#### Tutorial Updates
+ - Added a tip to the **Getting Started** page for downloading Desktop Apps.
+ - Updated tutorial icons and changed text content in tutorial tips.
+ - Updated the default treatment for the ``Add channel`` button to the current color and by team name.
+ - Added a tutorial tip for new settings and status buttons.
+ - Added a tip for the product switcher. This tip is skipped if not applicable.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added “Invite People” to the main "+" button below the hamburger menu.
+ - The whole category bounds are now highlighted while holding a channel above a category name on the left-hand side.
+ - Updated **Account Settings > Display > Timezone** to be more user friendly.
+ - New theme agnostic file preview modal takes up the full screen. The file preview now has information about the user, channel, and the file, and moves away from text-based buttons to icon-based buttons.
+ - Increased the limit of uploaded file attachments per post from 5 to 10.
+ - Added desktop notifications for followed Threads.
+ - Hungarian and English-Australian are now official languages.
+ - Added a **Start Trial** call-to action to the **Main Menu**.
+ - Changed H1-H3 heading font from Open Sans to Metropolis.
+
+#### Performance
+ - Improved typing performance when the emoji autocomplete is open.
+
+#### Integrations
+ - Dropped support for left-hand side-specific bot icons.
+ - Add a "rest field" to the App command parser.
+ - Added support for React components in channel header tooltips registered by plugins.
+ - Exported ``ChannelInviteModal`` and ``ChannelMembersModal`` components for plugins.
+ - Removed a deprecated "Backend" field from the plugin manifest.
+ - Converted the "Executables" field in the plugin manifest to a map.
+
+#### Administration
+ - Added ``playbooks`` and ``boards`` to restricted team URLs list.
```

Comment:
Do we want to mention what happens if a team currently uses these as URLs? (could be considered a breaking change)

Reply:
@jwilander ?

Reply:
They will generally continue to work but may break in some cases per https://docs.google.com/document/d/1Slo3ZWjy_uDIS3-vvzBkuT_7QRSQmVvEcxoYjf3g33E/edit#bookmark=id.n00o6dvxa7m2

Reply:
Let's note the cases that may break in the release notes